build.php

```diff
@@ -17,53 +18,66 @@
 //Add composer build, if flag --no-composer is undefined.
 //Dump autloader. 
 //Only if composer.json exists.
-if(file_exists('composer.json')) {
-    if(is_array($argv) && !in_array('--no-composer', $argv)) {
-        $buildCommands[] = 'composer install --prefer-dist --no-progress --no-dev'; 
+if (file_exists('composer.json')) {
+    if (is_array($argv) && !in_array('--no-composer', $argv)) {
+        $buildCommands[] = 'composer install --prefer-dist --no-progress --no-dev';
     }
+
     $buildCommands[] = 'composer dump-autoload';
 }
 
 //Run npm if package.json is found
-if(file_exists('package.json') && file_exists('package-lock.json')) {
-    $buildCommands[] = 'npm ci --no-progress --no-audit';
-} elseif(file_exists('package.json') && !file_exists('package-lock.json')) {
-    $buildCommands[] = 'npm install --no-progress --no-audit';
-}
-
-//Run build if package-lock.json is found
-if(file_exists('package-lock.json') && !file_exists('gulp.js')) {
-    $buildCommands[] = 'npx --yes browserslist@latest --update-db';
-    $buildCommands[] = 'npm run build';
-} elseif(file_exists('package-lock.json') && file_exists('gulp.js') && is_array($argv) && in_array('--allow-gulp', $argv)) {
-    $buildCommands[] = 'gulp';
+if (file_exists('package.json') && file_exists('package-lock.json')) {
+    if (is_array($argv) && !in_array('--install-npm', $argv)) {
+        $buildCommands[] = 'npm ci --no-progress --no-audit';
+        $buildCommands[] = 'npm run build';
+    } else {
+        $npmPackage = json_decode(file_get_contents('package.json'));
+        $buildCommands[] = "npm install $npmPackage->name";
+        $buildCommands[] = "rm -rf ./dist";
+        $buildCommands[] = "mv node_modules/$npmPackage->name/dist ./";
```

Comment:
Curious about this, what is te purpose of this?

Reply:
I'm suspecting that this is a typo of what exists in Municipio's build.php script;

```
$buildCommands[] = "rm -rf ./assets/dist";
$buildCommands[] = "mv node_modules/$npmPackage->name/assets/dist ./assets/";
```

Notice this does the same, though only with assets.

I'm investigating where this came from, as I do not recall writing the typo.